lib/DataFileUtil/DataFileUtilImpl.py

```diff
@@ -376,24 +376,28 @@ def _retrieve_filepath(self, file_url, cookies=None):
         _retrieve_filepath: retrieve file name from download URL and return local file path
 
         """
-
+        self.log('Retrieving file name from url: {}'.format(file_name))
         try:
-            with requests.get(file_url, cookies=cookies, stream=True) as response:
```

Comment:
Changed streaming GET request to a HEAD request since we're only interested in `Content-Disposition`. Don't need a context manager here